actuator.md

```diff
@@ -0,0 +1,15 @@
+# Actuator
+Actuator endpoints let you monitor and interact with DSpace. By default the following endpoints are enabled:
+
+**info**: Displays arbitrary application info. Only administrators can access this endpoint. 
```

Comment:
Change `info` to be the entire endpoint, as that makes it easier to understand how to use this.  So, change to `**/actuator/info**:`